etna/api/urls/events.py

```diff
@@ -0,0 +1,13 @@
+from etna.api.filters import LocationFilter
+from etna.api.urls.pages import CustomPagesAPIViewSet
+from etna.whatson.models import EventPage
+
+
+class EventsAPIViewSet(CustomPagesAPIViewSet):
+    known_query_parameters = CustomPagesAPIViewSet.known_query_parameters.union(
+        ["online", "at_tna"]
+    )
+
+    filter_backends = [LocationFilter] + CustomPagesAPIViewSet.filter_backends
+
```

Comment:
Should we only allow the listing path? We don't need the details path (because we can just use `/pages`) or the find path (we don't need to find events):

```suggestion
    @classmethod
    def get_urlpatterns(cls):
        return [
            path("", cls.as_view({"get": "listing_view"}), name="listing"),
        ]
```